src/argilla/client/feedback/dataset/base.py

```diff
@@ -119,49 +119,43 @@ def allow_extra_metadata(self) -> bool:
         """Returns whether if adding extra metadata to the records of the dataset is allowed"""
         return self._allow_extra_metadata
 
+    def __get_property_by_name(self, item_name: str, iterable_items, item_type: str):
+        for item in iterable_items:
+            if item.name == item_name:
+                return item
+            if hasattr(item, "title"):
+                if item.title == item_name:
```

Comment:
@frascuchon, I added this in the assumption that names can't overlap with titles. Is that correct?